.github/workflows/statistics-daily.yml

```diff
@@ -55,6 +55,15 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         shell: bash
         run: |
+          # keep in sync with the array in statistics-weekly.yml
+          declare -A lookupTeamMedic
+          lookupTeamMedic["Core Features"]="@core-features-medic"
+          lookupTeamMedic["Data Layer"]="@data-layer-medic"
+          lookupTeamMedic["Identity"]="@identity-medic"
+          lookupTeamMedic["General"]="General Test flakiness, requires investigation"
```

Comment:
🔧 Align the team "names" to GH/slack handles for team names for consistency:
- `Core Features` -> `team-core-features`
- `Data Layer` -> `team-data-layer`